datalad_dataverse/create_sibling_dataverse.py

```diff
@@ -149,6 +157,16 @@ class CreateSiblingDataverse(Interface):
             together, a publication dependency on the storage sibling is
             configured for the regular sibling in the local dataset clone.
             """),
+        collection=Parameter(
+            args=("--collection",),
+            constraints=EnsureStr() | EnsureNone(),
+            doc="""TODO
+
+            I think this likely needs to become several options to specify a
+            mode of operation (at least 'use an existing' vs 'create one') and
+            the respective specification.
```

Comment:
It is unlikely that a datalad user will have the power to create a collection. I think collections are typically few in number, and specific a dataverse instance (rather than a "user thing")